src/azure-cli/azure/cli/command_modules/acs/decorator.py

```diff
@@ -35,6 +50,8 @@
     _add_role_assignment,
     _ensure_aks_acr,
     create_load_balancer_profile,
+    _ensure_default_log_analytics_workspace_for_monitoring,
```

Comment:
sort it?

Reply:
Good point, changed in commit [0e7d1b9](https://github.com/Azure/azure-cli/pull/19478/commits/0e7d1b97b375241cccbf6b402de46c4c18e0076b).